test/image/mocks/gl3d_cone-single.json

```diff
@@ -0,0 +1,27 @@
+{
+  "data": [
+    {
+      "type": "cone",
+      "name": "blue cone",
+      "x": [1], "y": [1], "z": [1],
+      "u": [1], "v": [1], "w": [0],
+      "colorscale": "Blues",
+      "showscale": false
+    },
+    {
+      "type": "cone",
+      "name": "green cone",
+      "x": [3], "y": [3], "z": [3],
+      "u": [0], "v": [0], "w": [2],
+      "colorscale": "Greens",
+      "showscale": false
+    }
```

Comment:
Just to play devil's advocate re: autoscale - perhaps the single-cone mock should really be just one single cone, so the data has zero range without considering the cone itself? And multi-trace could be incorporated into another mock?

Reply:
yeah good call -> https://github.com/plotly/plotly.js/pull/2647/commits/7da71ae44f0297535ebd26049f76a21292e37fed